### PR TITLE
SorrTask369_Convert_Reading_List_gDoc_to_Markdown

### PR DESCRIPTION
**Pull request by hhxjqm**
#369

### CODE REVIEW

File: docs/Reading_List.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- [Bash / Linux](#bash--linux)
+- [Coding](#coding)
+- [Data analysis](#data-analysis)
+- [SRE](#sre)
+
+<!-- tocstop -->
+
+# Git
+
+- A [short tutorial](https://git-scm.com/docs/gittutorial)
```
Comment (DanilYachmenev) [inline docs/Reading_List.md:15]: Let's remove `A` from here
just leave a `Short tutorial` hyperlink

File: docs/Reading_List.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- [Pro Git book](https://git-scm.com/book/en/v2)
+  - To achieve mastery
+
+# Bash / Linux
+
+- A
+  [short tutorial](https://www.usenix.org/sites/default/files/conference/protected-files/lisa19_maheshwari.pdf)
```
Comment (DanilYachmenev) [inline docs/Reading_List.md:22]: Same here, just `Short tutorial`

File: docs/Reading_List.md
Comment (DanilYachmenev) [inline docs/Reading_List.md:1]: Lets put a reference to this doc to `docs/README.md` in `Learn how to become efficient at developing` section right under `Docker` bullet point`

**Review by DanilYachmenev — COMMENTED** [review]
all good, couple nits, 1 ref, and done

File: docs/Reading_List.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+- [Bash / Linux](#bash--linux)
+- [Coding](#coding)
+- [Data analysis](#data-analysis)
+- [SRE](#sre)
+
+<!-- tocstop -->
+
+# Git
+
+- [short tutorial](https://git-scm.com/docs/gittutorial)
```
Comment (DanilYachmenev) [inline docs/Reading_List.md:15]: Start with capital letter for consistency with other parts of the doc
same below

**Review by DanilYachmenev — COMMENTED** [review]
lgtm

File: docs/Reading_List.md (hunk 3 above)
Comment (DanilYachmenev) [reply docs/Reading_List.md]: @hhxjqm press `Resolve conversation` after you implement requested change

File: docs/Reading_List.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+# Git
+
+- [Short tutorial](https://git-scm.com/docs/gittutorial)
+- [Pro Git book](https://git-scm.com/book/en/v2)
+  - To achieve mastery
+
+# Bash / Linux
+
+- [Short tutorial](https://www.usenix.org/sites/default/files/conference/protected-files/lisa19_maheshwari.pdf)
```
Comment (gpsaggese) [inline docs/Reading_List.md:21]: Can you add a line about

[Missing semester of CS](https://missing.csail.mit.edu/)

in the bash section?

**Review by samarth9008 — COMMENTED** [review]
LG!

**Pull request merged**